common/models/types/index.d.ts

```diff
@@ -52,13 +69,11 @@ declare interface LexiconTraversal {
 
   /**
    * Allows direct access to the traversal state that results when appending a
-   * `char` representing a single UTF-16 codepoint to the current traversal
-   * state's prefix.  This bypasses the need to iterate among all legal child
-   * Traversals.
+   * `char` representing one or more individual UTF-16 codepoints to the
+   * current traversal state's prefix.  This bypasses the need to iterate
+   * among all legal child Traversals.
```

Comment:
Ref https://github.com/keymanapp/keyman/pull/11869#discussion_r1666328381. Probably even more important to be clear!